git_machete/client.py

```diff
@@ -1906,8 +1906,9 @@ def __sync_before_creating_pr(self) -> None:
         if current_branch not in self.managed_branches:
             self.add(current_branch)
 
-        if self.__is_merged_to_upstream(current_branch):
-            return
+        if self.__git.is_ancestor_or_equal(current_branch, self.up_branch.get(current_branch)):
+            raise MacheteException(
+                'All commits in head branch for this pull request are already included in base branch.\nCannot create pull request.')
```

Comment:
I'd include head and base branch names explicitly in the error message for clarity